### ISSUE TITLE
[Enhancement] Add compatible versions in the "More Information" box

### ISSUE THREAD

**Issue by B00ze64** [mozilla/addons-frontend#3507]
Good day.

Trying out new AMO via the "Use Mobile Site" link at the bottom of classic AMO.

Noticed that the "Version Information" pane, near the bottom of an AddOn's page in Old AMO is no longer there in the New AMO. This information is now listed in the "More Information" box just below the AddOn title, on New AMO, but it does not show for which version of Firefox the current version of the AddOn is compatible, which is information I could see in the old "Version Information" pane at the bottom. We now have to click on "See All Versions" to see this (which takes us back to Old AMO).

Suggest this is important information and should be shown in the new "More Information" box.

Thank you.
Best Regards,

**Comment by tofumatt** [comment]
Because new extensions are moving to WebExtension technology we forsee a lot fewer compatibility issues with add-ons. If an add-onis *incompatible* there will be notices on the page:

<img width="1228" alt="screenshot 2017-10-14 09 08 30" src="https://user-images.githubusercontent.com/90871/31574062-4b5df42c-b0bf-11e7-978b-25f7fae423f6.png">

But in general I don't think we want to list explicit version compatibility because we're hoping it will be less of an issue.

Please let me know if that's not clear or I missed something by commenting or re-opening the issue, but thanks for reporting it and trying out the new site. Let us know if there's anything else we should know about, we appreciate the issues 😄

**Comment by B00ze64** [comment]
@tofumatt I knew you were going to say this. The WE API will evolve, extensions will be updated and become incompatible with older Firefox's (e.g. ESR) while some extensions will stop being updated and will become incompatible with newer Firefox's. We need compatibility information to make decisions - The banner you listed does not explain if an extension is not compatible because it is too old, or because it is too new. Will you at least keep this information visible if we click on "See All version" ?

**Comment by tofumatt** [comment]
The banner does mention that a newer version is required if your browser is too old (https://github.com/mozilla/addons-frontend/blob/master/src/amo/components/AddonCompatibilityError/index.js#L73). But you're right: looks like the error message for your browser being too new is unclear. I recall at the time it was suggested to avoid the word "legacy" in the error message but clearly it has to be clearer somehow.

If we're going to show detailed compatibility info I'd say:

* the all versions place would be the place to do it
* we really only need to show it if the add-on *isn't* compatible

Would that work for you? I'd rather keep compatibility info out of the More Info box, just to keep from overloading casual users with info 😄

**Comment by B00ze64** [comment]
@tofumatt Hi! I'm OK with the banner being a generic "won't work for you" so long as we have compatibility information _somewhere_. The All Versions place looks indeed like the right place for it. I would go for showing it there no matter what - I often make decisions based on compatibility. It's hard to give good examples but I pretty much always hit the "Show All Versions" and look at why an addOn has been updated when I am several versions behind. The Compatible-With gives some additional information, especially when an author does not publish any version history notes. It's also happened often that I was wondering why an addOn I had was not updating to the latest version until I saw, because of Compatible-With, that I had a too old Firefox. Thanks!
